lambdas/services/manifest_dynamo_service.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+from enums.supported_document_types import SupportedDocumentTypes
+from services.dynamo_service import DynamoDBService
+from enums.metadata_field_names import DocumentReferenceMetadataFields
+from models.document import Document
+from utils.exceptions import DynamoDbException
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class ManifestDynamoService(DynamoDBService):
+
+    def discover_uploaded_documents(
+            self, nhs_number: str, doc_type: SupportedDocumentTypes
+    ) -> list[Document]:
+        documents = []
+        if doc_type == SupportedDocumentTypes.ARF:
+            document_table = os.environ["DOCUMENT_STORE_DYNAMODB_NAME"]
+        elif doc_type == SupportedDocumentTypes.LG:
+            document_table = os.environ["LLOYD_GEORGE_DYNAMODB_NAME"]
```

Comment:
We're already checking document type when we're calling this 'discover upload documents', can we get the table name in the lambda and pass it in to this function instead of checking the type again?

Reply:
Good catch, I've made some changes